src/reports/filters.php

```diff
@@ -47,7 +48,13 @@ public function get()
 		}
 
 		$reportPlugin = new $className;
-		$filters      = $reportPlugin->displayFilters();
+
+		$filters = $reportPlugin->displayFilters();
+
+		foreach ($filters[0] as $key => $value) {
+			$value['name'] = $key;
+			$filters[0][$key] = $value;
```

Comment:
There should be no key. We just need a normal array, not an associative array.

Reply:
done